datahub-web-react/src/app/domain/DomainItemDelete.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { DeleteOutlined } from '@ant-design/icons';
+import { Dropdown, Menu, message, Modal } from 'antd';
+import { EntityType } from '../../types.generated';
+import { useEntityRegistry } from '../useEntityRegistry';
+import { useDeleteDomainMutation } from '../../graphql/domain.generated';
+import { MenuIcon } from '../entity/shared/EntityDropdown/EntityDropdown';
+
+type Props = {
+    urn: string;
+    name: string;
+    onDelete?: () => void;
+};
+
+export default function DomainListItem({ name, urn, onDelete }: Props) {
```

Comment:
Name of the file should be DomainListItem I think :)

Reply:
Or, on closer loop, this actually this should probably be DomainItemDelete

Reply:
Actually - do you mind if we leave this name as is? Why change it?

In the future, we will likely add more options here.

Reply:
I changed it to `DomainItemMenu`, I was developing this when I had the previous implementation present so that's why I created a new file. I hope the current approach is ok though.